### PR TITLE
DeepLabv3+ README

### PR DESCRIPTION
**Pull request by Landanjs**
Adds DeepLabv3+ README

### CODE REVIEW

File: composer/models/deeplabv3/README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+Based on [Encoder-Decoder with Atrous Separable Convolution for Semantic Image Segmentation](https://arxiv.org/abs/1802.02611)
+
+<div align=center>
+<img src="https://storage.googleapis.com/docs.mosaicml.com/images/models/deeplabv3_v2.png" alt="deeplabv3plus" width="650">
+</div>
+
+
+- **Backbone network**: converts the input image into a feature map.
+    * Usually ResNet-101 with the strided convolutions removed in stage 3 and 4.
```
Comment (A-Jacobson) [inline composer/models/deeplabv3/README.md:29]:
```suggestion
    * Usually ResNet-101 with the strided convolutions converted to dilations convolutions in stage 3 and 4.
```

previous wording suggested the conv layers were actually removed.

File: composer/models/deeplabv3/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    * The 3x3 convolutions in stage 3 and 4 have dilation sizes of 2 and 4, respectively, to compensate for the decreased receptive field.
+    * The average pooling and classification layer are ignored.
+- **Spatial Pyramid Pooling**: extracts multi-resolution features from the backbone feature map.
```
Comment (A-Jacobson) [inline composer/models/deeplabv3/README.md:32]:
```suggestion
- **Spatial Pyramid Pooling**: extracts multi-resolution features from the stage 4 backbone feature map.
```

**Review by A-Jacobson — APPROVED** [review]
tiny clarification on the architecture. the rest lgtm.

File: composer/models/deeplabv3/README.md (hunk 1 above)
Comment (Landanjs) [reply composer/models/deeplabv3/README.md]: Sounds good!

**Comment by jfrankle** [comment]
I know I'm behind, but LGTM

**Pull request merged**